### ISSUE TITLE
How to improve the training accuracy as much as possible?

### ISSUE THREAD

**Issue by tms2003** [Peterande/D-FINE#7]
I found that the current instructions have multiple different weights, including coco, coco+365, and 365 pre-trained. There is also a default weight (automatically downloaded during training). I want to train on my own dataset, so what training parameters or tuning parameters should I use? Should I start with the 365 pre-trained weights or the coco+365 pre-trained weights?and how many epoch is better?

**Comment by Peterande** [comment]
Thank you for your question!

- **Model Tuning**:  
   The best approach is to use the pretrained model with the suffix `obj365` for tuning.

- **Class Mapping**:  
   To further retain the pretrained capabilities, you can follow the steps in *Usage → Custom Dataset → Modify Class Mappings* to map the Objects365 categories to your dataset.  
   - If there are no duplicate categories, or you don't mind convergence speed differences, you can skip this step.

- **Epochs and Hyperparameters**:  
   You need to test the epochs and hyperparameters based on your dataset, as fine-tuning a pre-trained model on a small dataset may lead to overfitting.
   - My suggestion: Set a relatively large number of epochs and observe the convergence.
   - Adjust the number of epochs for further training based on the results.

- **Config Update**:  
   I have already added configurations for *Tuning on Custom Dataset* and updated the README with the steps in *Usage → Custom Dataset → Tuning on Custom Dataset*.

Thank you for your support of **D-FINE**!
